src/components/instantlaunches/admin/InstantLaunchList.js

```diff
@@ -0,0 +1,432 @@
+        },
+    });
+
+    const isLoading = allILs.isLoading || dashboardILs.isLoading;
+    const isError = allILs.isError || dashboardILs.isError;
+
+    return (
+        <div width="100%">
+            {isLoading ? (
+                <Skeleton
+                    variant="rect"
+                    animation="wave"
+                    height={300}
+                    width="100%"
+                    id={skeletonID}
+                />
+            ) : isError ? (
+                <WrappedErrorHandler
+                    errorObject={allILs.error || dashboardILs.error}
+                    baseId={baseID}
+                />
+            ) : (
+                <>
+                    <CreationDialog
+                        t={t}
+                        open={dlgOpen}
+                        onClose={(event) => {
+                            event.stopPropagation();
+                            event.preventDefault();
+
+                            setDlgOpen(false);
+                        }}
+                    />
+
+                    <TableContainer className={classes.table} component={Paper}>
+                        <div className={classes.tableTitle}>
+                            <Typography variant="h5" component="span">
+                                {t("currentInstantLaunches")}
+                            </Typography>
+
+                            <Button
+                                variant="contained"
+                                color="primary"
+                                startIcon={<AddIcon />}
+                                className={classes.addButton}
+                                onClick={(event) => {
+                                    event.stopPropagation();
+                                    event.preventDefault();
+
+                                    setDlgOpen(true);
+                                }}
+                                id={buildID(baseID, ids.ADD, ids.BUTTON)}
+                            >
+                                {t("common:new")}
+                            </Button>
+                        </div>
+
+                        <Typography
+                            variant="body2"
+                            className={classes.tableDescription}
+                        >
+                            {t("listDescription")}
+                        </Typography>
+
+                        <Table>
+                            <TableHead>
+                                <TableRow>
+                                    <TableCell>{t("common:name")}</TableCell>
+                                    <TableCell>{t("createdBy")}</TableCell>
+                                    <TableCell>{t("addedOn")}</TableCell>
+                                    <TableCell></TableCell>
+                                    <TableCell></TableCell>
+                                </TableRow>
+                            </TableHead>
```

Comment:
We might want to move a lot of the components that we have in `components/utils` up and into the top-level `components` directory at some point. I've been skipping over the utils directory because the name doesn't exactly scream "look here for commonly used components". I was assuming it was just utility functions and stuff.